### PR TITLE
fix: tab bar icons, sizing and coloring

### PR DESCRIPTION
**Pull request by mikaelbr**
Updates Icons to the latest icons (small SVG changes), fixes coloring when active and text sizing.

Also sets max font size increase. Unsure if this is the best approach and most accessible. But feels it might be the most pragmatic (and also following WCAG guidelines). Open for input on this. See screenshots. Might solve #649 

Solves #666 

image

image

image

### CODE REVIEW

**Comment by rikkekantega** [comment]
We've actually already added 2 as max font multiplier for theme text. This is indeed WCAG compliant. I think further font scaling isn't what's going to improve accessibility the most. We should perhaps rather explore options like implementing pinch zoom and contrast theme in the future.

File: src/navigation/TabNavigator.tsx [hunk 1]
```diff
@@ -49,37 +52,55 @@ const NavigationRoot = () => {
...
+function tabSettings(
+  tabBarLabel: string,
+  Icon: (svg: SvgProps) => JSX.Element,
+): TabSettings {
+  return {
+    tabBarLabel: ({color}) => (
+      <Text
+        style={{color, fontSize: 14, lineHeight: 14}}
+        maxFontSizeMultiplier={2}
+      >
```
Comment (rikkekantega) [inline src/navigation/TabNavigator.tsx:100]: Should we perhaps consider using ThemeText component and type here instead?

Comment (mikaelbr) [reply src/navigation/TabNavigator.tsx]: You're right. I did this initially but found that the default lineHeight didn't work in this case, and ended up doing it manually. But at this point it's easier to use ThemeText and override line height manually. Updated

**Review by rikkekantega — APPROVED** [review]
Code LGTM :rocket:

**Review by hildeor — APPROVED** [review]

**Pull request merged**